src/main/java/org/jenkinsci/test/acceptance/plugins/gradle/GradleInstallation.java

```diff
@@ -33,7 +34,19 @@ public GradleInstallation(Jenkins context, String path) {
         super(context, path);
     }
 
+    public static final String LATEST_VERSION = "latestVersion";
+
     public static void installGradle(Jenkins jenkins, String name, String version) {
-        installTool(jenkins, GradleInstallation.class, name, version);
+        waitForUpdates(jenkins, GradleInstallation.class);
+
+        jenkins.getPluginManager().checkForUpdates();
+
+        ConfigurablePageObject tools = ensureConfigPage(jenkins);
+        ToolInstallation maven = addTool(jenkins, GradleInstallation.class);
+        maven.name.set(name);
+        if(!LATEST_VERSION.equals(version)) {
+            maven.installVersion(version);
+        }
```

Comment:
Can we adjust the base class? If there is no version given (null) to not call installVersion?
Or add another method that does not install a version.